onnxruntime/core/graph/ep_api_types.h

```diff
@@ -196,6 +199,9 @@ struct EpNode : public OrtNode {
   InlinedVector<EpValueInfo*> inputs_;
   InlinedVector<EpValueInfo*> outputs_;
 
+  std::unordered_map<std::string, std::unique_ptr<ONNX_NAMESPACE::AttributeProto>> attributes_map_;
```

Comment:
nit: is this `attribute_map_` used anywhere? Perhaps it could be used in the API that gets an attribute by name instead of searching through the attributes array.